build-config/conf/busybox.config

```diff
@@ -777,6 +777,9 @@ CONFIG_WATCHDOG=y
 CONFIG_SYS_EEPROM_I2C_DEVICE=""
 CONFIG_SYS_EEPROM_I2C_ADDR=
 CONFIG_SYS_EEPROM_I2C_MEM_ADDR_BITS=0
+CONFIG_SYS_EEPROM_I2C_WRITE_1B_DELAY=0
+CONFIG_SYS_EEPROM_I2C_WRITE_2B_DELAY=0
+CONFIG_SYS_EEPROM_I2C_WRITE_3B_DELAY=0
```

Comment:
I'm sorry, but I don't understand.  Under what circumstances do these values become "0" ?

I tried a couple of experiments:

1.  I tried with no busybox config.  I just unpacked the tarball, applied the patches and typed "make menuconfig" the busybox build directory with no pre-existing .config file present.  In this case I had to navigate the config menu and turn on ONIE system eeprom, which was expected.  Then I quit the menuconfig and saved the config.  The saved config had the default `[123]B_DELAY` values from misc/Config.src.  -- that all seems fine to me

2. another experiment -- I removed the `[123]B_DELAY` settings from our default config in build-config/conf/busybox.config.  I just removed the lines.  Then I "built" a machine that only enabled the ONIE syseeprom, without setting the `[123]B_DELAY` values.  That created a .config file that was just wrong -- the `[123]B_DELAY` values were not set at all.  That would lead to compile failures.  I think we should #error if these macros are unset in include/24cXX.h

3. another experiment -- the `[123]B_DELAY` values are set in our baseline config to their default values, like what I suggested up above.  This resulted in a build time busybox config that used the default values.  That seems correct to me.

One thing that came to mind during all this is perhaps we should set the `range` property on these configuration values.  For example:
```
config SYS_EEPROM_I2C_WRITE_2B_DELAY
        int "delay time for i2c writing 2 bytes"
        range 1 100000
        default 1000
        depends on SYS_EEPROM_DEVICE_I2C
        help
          Delay number of microseconds after writing 2 bytes.
```

This enforces the value is always greater than zero.